layers/raw.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+// Copyright 2009-2011 Andreas Krennmair. All rights reserved.
```

Comment:
this second copyright notice isn't necessary on new files, just the Google line will do.